config/mainnet.json

```diff
@@ -25,14 +25,6 @@
     }
   ],
   "savingsManagers": [
-    {
-      "address": "0xcEfCBb45BE41331337d49a131faF424D1e50234f",
```

Comment:
are we sure about removing these contracts?

Reply:
honestly I'm not sure, from my understanding these addresses are not used as in `protocol/subgraph.yaml` there's only 1 hard coded contract for SavingsManager which is on polygon testnet. My guess is when deploying to eth mainnet we will manually change the hard coded contract to one of these 3, and the latest one is `0x9781c4e9b9cc6ac18405891df20ad3566fb6b301` so I removed the rest, otherwise there will be error compiling